src/main/java/uk/gov/ida/verifyserviceprovider/VerifyServiceProviderApplication.java

```diff
@@ -24,19 +24,14 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(VerifyServiceProviderApplication.class);
 
-    private boolean fileSystemConfig;
-
-    private VerifyServiceProviderApplication(boolean fileSystemConfig) {
-        this.fileSystemConfig = fileSystemConfig;
-    }
-
     public VerifyServiceProviderApplication() {
-        this(true);
     }
 
     public static void main(String[] args) throws Exception {
         if (Arrays.asList(args).isEmpty()) {
-            new VerifyServiceProviderApplication(false).run("server", "verify-service-provider-env.yml");
+            String jarPath = VerifyServiceProviderApplication.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+            String configFilePath = jarPath.replaceFirst("/lib/verify-service-provider-.*\\.jar", "/verify-service-provider.yml");
```

Comment:
Any less horrendous suggestions for this are appreciated.

Reply:
.getParent doesn't appear to be a function...

Reply:
Oh, I see, VMSTT goes via creating a File object - I thought that seemed even messier, although happy to swap for consistency.